test/Constraints/protocols.swift

```diff
@@ -178,19 +178,19 @@ func existential(_ p: P) {
 }
 
 func staticExistential(_ p: P.Type, pp: P.Protocol) {
-  let _ = p() // expected-error{{initializing from a metatype value must reference 'init' explicitly}}
-  let _ = p().bar // expected-error{{initializing from a metatype value must reference 'init' explicitly}}
-  let _ = p().bar(1) // expected-error{{initializing from a metatype value must reference 'init' explicitly}}
+  let _ = p() // expected-error{{value of type 'P.Type' is a protocol; it cannot be instantiated}}
+  let _ = p().bar // expected-error{{value of type 'P.Type' is a protocol; it cannot be instantiated}}
+  let _ = p().bar(1) // expected-error{{value of type 'P.Type' is a protocol; it cannot be instantiated}}
```

Comment:
This diagnostic should stay the same, see my previous comment about difference between `P.Type` and `P.Protocol`.